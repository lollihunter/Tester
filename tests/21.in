20 20 20
1 1
16 12
16 13
15 8
8 15
7 2
2 6
1 17
16 7
8 6
17 4
15 16
19 13
5 17
15 4
6 16
1 18
15 2
14 6
20 20
