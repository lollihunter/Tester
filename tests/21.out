10
1 7 6 10 19 4 2 3 12 20 
